contracts/test/vault/index.js

```diff
@@ -33,12 +33,11 @@ describe("Vault", function () {
     const origAssetCount = await vault.connect(governor).getAssetCount();
     expect(await vault.isSupportedAsset(ousd.address)).to.be.false;
 
-    const latestBlock = await hre.ethers.provider.getBlock();
     /* Mock oracle feeds report 0 for updatedAt data point. Set
-     * maxStaleness to current block time + 1 day to make the Oracle
```

Comment:
This comment still sounds like staleness  this is an absolute time (from the unix epoch), rather than a relative time from the last update?

Reply:
Other than this, approved, and good to merge

Reply:
Heh so weird. I changed this already in a [commit before](https://github.com/OriginProtocol/origin-dollar/pull/1491/commits/8ad64de8a58cfc4de36347e111afc3890aec6eee) you've made a comment on it.

Puzzling how that could happen... Hmm maybe I've mad a commit and haven't pushed it yet to the branch. 🤷